lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -129,9 +150,32 @@ def get(self, request, *args, **kwargs):
             'course_tools': course_tools,
             'dates_widget': dates_widget,
             'handouts_html': handouts_html,
+            'welcome_message_html': welcome_message_html,
```

Comment:
I don't love this naming since it could technically be the latest update, but I don't have a better idea so oh well.

Reply:
Maybe `course_update_html` and using `course_update` language instead? Technically, the welcome message is just another upate so that might be more clear?